test/e2e/flowaggregator_test.go

```diff
@@ -212,6 +213,11 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 	})
 	require.NoError(t, err, "IPFIX collector did not receive expected number of data records and timed out.")
 
+	if checkBandwidth {
+		// Wait one more export interval of flow aggregator to calculate a more accurate bandwidth value
+		time.Sleep(5 * time.Second)
```

Comment:
could we use a const based on export value?

Reply:
Thanks, addressed.